pages/index.js

```diff
@@ -3,13 +3,13 @@ import Card from '../components/Card';
 
 export default function Home() {
   return (
-    <div className="container">
+    <div className="w-full">
       <Head>
         <title>Glitter</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <header className="flex flex-row justify-around pt-4 divide-x">
-        <div className="flex-none w-3/12">
+      <header className="flex place-items-center h-16 w-full justify-evenly">
+        <div className="flex justify-center w-3/12">
```

Comment:
The logo is not aligned with the menu. We want it aligned with the other items in the column.